docs/semantics-of-k.aux

```diff
@@ -0,0 +1,282 @@
+\relax 
```

Comment:
You should not include the auxiliary files in the repo!  Only include the PDF, so people do not have to compile the latex file in order to read the K semantics.